.evergreen/create-expansions.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+# Get the current unique version of this checkout.
+if [[ "$is_patch" = true ]]; then
+  CURRENT_VERSION=$(git describe)-patch-${version_id}
+else
+  CURRENT_VERSION=latest
+fi
+
+DRIVERS_TOOLS="$(pwd)/../drivers-tools"
+
+# Python has cygwin path problems on Windows. Detect prospective mongo-orchestration home
+# directory.
+if [[ "${OS}" = "Windows_NT" ]]; then
+  export DRIVERS_TOOLS=$(cygpath -m $DRIVERS_TOOLS)
+fi
+
+MONGO_ORCHESTRATION_HOME="${DRIVERS_TOOLS}/.evergreen/orchestration"
+MONGODB_BINARIES="${DRIVERS_TOOLS}/mongodb/bin"
+UPLOAD_BUCKET="${project}"
+PROJECT_DIRECTORY="$(pwd)"
+LIBMONGOCRYPT_SUFFIX_DIR="lib"
```

Comment:
Let's prune this file down to just what is needed.  We don't need anything about `LIBMONGOCRYPT` for example.